### PR TITLE
OCM-5853: exclude commits validations for PRs created by specific authors

### PR DESCRIPTION
**Pull request by nirarg**
**What this PR does / why we need it**:
This is done for exclude commits in auto generated PRs Now added commit message from red-hat-konflux

**Which issue(s) this PR fixes**:
Fixes #[OCM-5853](https://issues.redhat.com//browse/OCM-5853)

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Build
- [x] CI
- [ ] Documentation
- [ ] Performance
- [ ] Refactor
- [ ] Style
- [ ] Unit tests
- [ ] Subsystem tests

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

**Review by enriquebelarte — APPROVED** [review]

File: .github/workflows/check-commit-format.yml [hunk 1]
```diff
@@ -20,18 +20,32 @@ jobs:
         env:
           BASE_SHA: ${{ github.event.pull_request.base.sha }}
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
+          PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+
         run: |
           IFS=$'\n' commit_messages=($(git log --pretty=format:"%s" $BASE_SHA...$HEAD_SHA))          
           
+          declare -a skip_pr_authors=(
+            "red-hat-konflux"
```
Comment (bardielle) [inline .github/workflows/check-commit-format.yml:29]: add a comment that this is an RHTAP user please

Comment (enriquebelarte) [reply .github/workflows/check-commit-format.yml:29]: It's OK to add the comment for clarification purposes but RHTAP is now named Konflux:  
"Simultaneously, we will continue to work on the stack _formerly known as RHTAP_ by forming a new upstream under the name **Konflux**."

Comment (nirarg) [reply .github/workflows/check-commit-format.yml:32]: @enriquebelarte thank you for the details
Anyway, I added a comment

**Pull request merged**